src/ansys/mapdl/core/pool.py

```diff
@@ -93,7 +93,7 @@ class LocalMapdlPool:
         By default, the instances directories are named as "Instances_{i}".
 
     **kwargs : dict, optional
-        See :func:`ansys.mapdl.core.launch_mapdl` for a complete
+        See :func:`ansys.mapdl.core.launcher.launch_mapdl` for a complete
         listing of all additional keyword arguments.
```

Comment:
```suggestion
        Additional keyword arguments. For a complete listing, see the description for the
        :func:`ansys.mapdl.core.launcher.launch_mapdl` method.
```
A brief description should always be the first thing provided for a parameter.